feat(geo): update Geo to reflect React component release (#4190)
aws-amplify/docs: src/fragments/lib/geo/js/maps.mdx

First, ensure you've provisioned an Amazon Location Service Map resource and configured your app using the instructions in either [Amplify CLI - Geo - Maps](/cli/geo/maps) or [Use existing resources](/lib/geo/existing-resources) guide.

<Callout>

> **Note:** If you are using React, you should use the [Amplify React MapView component](https://ui.docs.amplify.aws/components/geo)

Reviewer: ```suggestion
**Note:** For React, you can use the [Amplify React MapView component](https://ui.docs.amplify.aws/components/geo)
```
